changelogs/fragments/74986-ansible-galaxy-playbook-dir.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
```

Comment:
This should probably be minor_changes since it's featurish (new user-facing option).

Reply:
yes, this is new option, not bugfix

Reply:
done